.github/workflows/test.yml

```diff
@@ -181,7 +181,7 @@ jobs:
         uses: actions/setup-node@v4
         with:
           # Match workers-sdk node version
-          node-version: 18.20.5
+          node-version: 20.19.1
```

Comment:
Can you update this to v22? We use v22 internally.

https://github.com/cloudflare/workerd/blob/e94e9d689591d02b4fe364d1a0c76bca0bc09335/WORKSPACE#L21

Reply:
the comment says:

> # Match workers-sdk node version

Do you disagree with that?

Reply:
If the goal is to avoid breaking wrangler's CI with workerd commits, this makes the most sense I'd say.

Reply:
Yeah, I'd vote for keeping this at 20. These tests also run with Node 22 before a workers-sdk release